### ISSUE TITLE
Pictures zoom factor ratio changed, some pictures became useless

### ISSUE THREAD

**Issue by Githopp192** [nextcloud/cookbook#216]
When i began using this great app (thank you very much for developping) .. sometimes i just took some photos from my personal cookbook ( in paper ) .. and uploaded them to the Nextcloud cookbook app.

and then it was also possible to set keywords within a recipe, which then served as categories.

Even though you can now make categories manually (great thing) ... but all my previous categories (created by the keywords) are gone.

And just ... some of the pictures in the app only show an enlarged section. Zooming within this picture is very tedious and I almost have to say that all my recipe pictures (more than 150) have become almost useless :-( 

Thanks for your support. i do appreciate

**Comment by mcorteel** [comment]
The keyword to category move is already in discussion somewhere else (#188).

As for images, you can click on the image to display the full image. They are only cropped to save vertical space by default.
If that answers your image problem, then we can close this issue.

**Comment by Githopp192** [comment]
yes .. just i say it (#188) .. good to know, that there is a discussion behind :-) and it would really help so that you could categorize by keywords (the way it used to work).

As for images .. only click to see the full image .. this does not work anymore ..(i tried this with different browsers .. at the moment with the new Edge) .. 

Yes .. that the images now were cropped .. is an issue. It would not be an issue .. if the picture would only a picture and the recipe would be stored in the individual fields.
(which would actually correspond to the admittedly intended purpose of the application :-) ) 

But because I have many recipes only as pictures, I have a problem now.

Screenshot 1: shows the picture on web interface
Screenshot 2: shows the picture when I make the first click
--> no display of the whole picture
Screenshot 3: shows the picture when I make the third click
--> The picture is enlarged again ... but I have no possibility to read the recipe (e.g. by moving the picture).

So .. at the moment i've got no chance to read my recipes.

Optimal would be for me, if it would be like in one of the pre-releases.
![ScreenShot_3](https://user-images.githubusercontent.com/32612067/81051278-bed9d280-8ec1-11ea-99bd-190aeace5d8b.jpg)
![ScreenShot_2](https://user-images.githubusercontent.com/32612067/81051282-c00aff80-8ec1-11ea-84b5-85aa1c19b8f3.jpg)
![ScreenShot_1](https://user-images.githubusercontent.com/32612067/81051283-c0a39600-8ec1-11ea-925a-85aeca3086ab.jpg)

**Comment by mrzapp** [comment]
@mcorteel @Githopp192 The problem seems to be that images that are too large for the screen are not accommodated very well, they just sort of stick out to the right. I'll make a temporary patch for this, while we await the completeion of `0.7.0`

**Comment by Githopp192** [comment]
Thanks ... that would be great and a great help. I wish you further success developping the app

**Comment by Githopp192** [comment]
@mcorteel ... this issue re-occurs again - cannot size-/rezsize the Images anymore.

**Comment by mcorteel** [comment]
@Githopp192 I haven't worked on this project in a year (and it has essentially been rewritten from scratch). You should turn to a maintainer: @mrzapp maybe ?

**Comment by seyfeb** [comment]
@Githopp192 What versions of NC and the Cookbook app are you using?

**Comment by christianlupus** [comment]
@Githopp192 I tried to reproduce this issue with the current version and I have a problem reproducing it. I did the following steps:
1. Take any A4 PDF with 1 page (I just generated one)
2. Make a JPG file out of it (I used `pdttoppm File.pdf | pnmtojpg > File.jpg` under Linux)
3. Upload the file to my test instance
4. Create a new recipe with title, dummy description (one word) and the image I just uploaded. Saving recipe.
5. Clicking on the image brings it to full height and smaller (as before) alternatingly.

Please make sure, you are on a recent version of cookbook. Please share the width of your browser window in pixels as the visual representations changes with the width of the monitor (e.g for mobile phone and tablet usage).

Please also be a bit more verbose about the issue. What device are you working at? Can you share screenshots?

**Comment by Githopp192** [comment]
Hey Christian,

nice to hear from you and again many thanks for this great App (you may already know ..it's one of my favorites :-) 

currently i'm using NC Version: 20.0.6, cookbook: 0.8.4

I've got this issue again, on every device (iPad,iPhone,Edge-/Chrome-/Firefox Browser).

Here's just one example of two pics (1rst without a mouse click into the pic, the other with a mouse click on the pic).

![ScreenShot079](https://user-images.githubusercontent.com/32612067/113448386-092e4100-93fc-11eb-9309-989660725a59.jpg)
![ScreenShot114](https://user-images.githubusercontent.com/32612067/113448417-164b3000-93fc-11eb-9c69-48920ad58dca.jpg)

Previously i was able to resize the Pics on every device.
(Imagine ..often i'm cooking with the iPad in front of me - trying to resize the Pics .. when this is not possible,then 
i'd need to make normal picture (camera); from there i can resize the recipe, that i can read or scrolll up/down - 
for me this is crucial, because most of my recipes are pics with a short reference to the cookbook only (page).

2nd example:

just choose one recipe (logged in via Safari Browser on my iPhone (IOS 14.4.2/iPhone SE 2020).

PIC 1 - without a mouseclick
![ScreenShot115](https://user-images.githubusercontent.com/32612067/113449400-0f252180-93fe-11eb-9f92-80dc2b4fdc80.jpg)
PIC 2 with one mouseclick
![ScreenShot116](https://user-images.githubusercontent.com/32612067/113449429-18ae8980-93fe-11eb-96a1-c4c5abc03d14.jpg)

3rd example (here it looks a little bit different - in that way, that 1rst pic is small (as it should be probably), and 2nd Picture will be resized to full size (on every device - like iPhone, Windows 10..)


![ScreenShot117](https://user-images.githubusercontent.com/32612067/113449863-08e37500-93ff-11eb-9701-a7e2e9ed4491.jpg)
Resized to full pic size: 

![ScreenShot118](https://user-images.githubusercontent.com/32612067/113449882-13057380-93ff-11eb-92ad-9ab5ef449959.jpg)

But the issue is .. previously i was able to resize the picture (when clicking on it) - either through mouse scrolling (Windows 10/Edge/Firefox/Chrom) or ipad/iPhone (Safari/Edge/Firefox) (--> finger spreading).

This is not possible anymore.

This are the box/padding information from the last example.

1rst mouseclick:

![ScreenShot119](https://user-images.githubusercontent.com/32612067/113450423-341a9400-9400-11eb-9380-fef0dac949a2.jpg)

2nd Mouseclick:

![ScreenShot120](https://user-images.githubusercontent.com/32612067/113450452-4399dd00-9400-11eb-80d3-1e756d0f0bb1.jpg)

**Comment by christianlupus** [comment]
OK, now I get the issue but it is independent of the previous problems. This was introduced by the CSS effort to make it work on any device (width) in a responsive manner. Zooming on the image was not on the agenda back then. So in the portrait mode, it is fixed to `width: 100%` currently for example.
